repositories/repositories.bzl

```diff
@@ -131,6 +131,14 @@ def ros2_workspace_repositories():
         urls = ["https://github.com/zaphoyd/websocketpp/archive/refs/tags/0.8.2.tar.gz"],
     )
 
+    maybe(
+        http_archive,
+        name = "with_cfg.bzl",
+        sha256 = "d83f99ac39cd9940848ea11a51a60159cf09cda2ba30545036041551aae73ab4",
+        strip_prefix = "with_cfg.bzl-0.6.1",
+        url = "https://github.com/fmeum/with_cfg.bzl/releases/download/v0.6.1/with_cfg.bzl-v0.6.1.tar.gz",
```

Comment:
yes this isn't the latest release, but newer ones seem to cause issues with struct members of `@bazel_features` not being available when building in Workspace mode